brewtils/rest/system_client.py

```diff
@@ -203,17 +203,46 @@ def __init__(self, *args, **kwargs):
             )
             kwargs.setdefault("system_name", args[2])
 
-        self._system_name = kwargs.get("system_name")
-        self._version_constraint = kwargs.get("version_constraint", "latest")
-        self._default_instance = kwargs.get("default_instance", "default")
+        system_kwargs = {
```

Comment:
This isn't really a list of system kwargs anymore, it's more like a map of kwarg names to config item names. So let's give it a different name, maybe something like `config_map`?